Gaijinizer patch/Map047.txt

```diff
@@ -4,109 +4,111 @@
 
 道の先になにかいるぞ。
 > CONTEXT: Map047/events/38/pages/0/8/Dialogue/yuuma
-Wait, \\N[1]. I have something to do ahead of the
-road.#MTLed
+Wait, \\N[1]. There's something ahead of you.#Ron
 > END STRING
 
 > BEGIN STRING
 な、なんだあれ…。ここじゃ後ろ姿しか見えないけど……。
 
 あれも『悪魔』なのか…？
 > CONTEXT: Map047/events/38/pages/0/15/Dialogue/kazuya
-Well, what? I can only see the back view here ...
-Is that "demon" ...?#MTLed
+W- what the hell's  that? I can only see its back,
+Is it a demon...?#Ron
 > END STRING
 
 > BEGIN STRING
 ああ、あれは『マザーラビット』だな。
 
 よりによって、面倒なやつと鉢合わせしてしまったか…
 > CONTEXT: Map047/events/38/pages/0/19/Dialogue/yuuma
-Oh, that's 'Mother Rabbit'. By the way, have you
-gotta be bothered with a troublesome guy ...#MTLed
+Looks like a 'Mother Rabbit'. Of all the damn
+things to run into...#Ron
 > END STRING
 
 > BEGIN STRING
 なんだよ、その言い方…。
 
 そ、そんなにヤバイやつなのか？あいつ。
 > CONTEXT: Map047/events/38/pages/0/23/Dialogue/kazuya
-That's how it says .... So, are you so bad guys?
-you.#MTLed
+What're you talking about? Is that thing really
+that dangerous?#Ron
 > END STRING
 
 > BEGIN STRING
 いや……危害は加えてこない。
 
 バーゲスト同様、むしろそれが問題なんだ。
 > CONTEXT: Map047/events/38/pages/0/27/Dialogue/yuuma
-No...There is no harm. Like barghest, it's
-rather a problem.#MTLed
+No... It's not particularly dangerous, it's like
```

Comment:
It feels like it'd flow better if you rearranged the middle sentence...
>_**No... Like the Barghests, it's not particularly dangerous.
 In fact, that's the problem...**_